galaxy_ng/app/migrations/0011_move_collections_by_certification.py

```diff
@@ -0,0 +1,87 @@
+from django.db import migrations
+
+MIGRATE_COLLECTIONS_QUERY = """
+CREATE FUNCTION _move_collection_versions_by_certification(
+    from_repo_name TEXT, to_repo_name TEXT,  filter TEXT
+) RETURNS void AS
+$$
+DECLARE
+    from_repo UUID := (
+        SELECT pulp_id FROM core_repository WHERE name = from_repo_name
+    );
+    to_repo UUID := (
+        SELECT pulp_id FROM core_repository WHERE name = to_repo_name
+    );
+    from_version UUID;
+    to_version   UUID;
+BEGIN
+    INSERT INTO core_repositoryversion (
+        pulp_id, pulp_created, pulp_last_updated, number, 
+        complete, base_version_id, repository_id
+    ) VALUES (
+        uuid_generate_v4(), now(), now(), 
+        (SELECT max(number) + 1 AS number FROM core_repositoryversion 
```

Comment:
You should use [next_version on core_repository](https://github.com/pulp/pulpcore/blob/4063c346b4b75ffe74d2b0a95d650732dfb5ed8e/pulpcore/app/models/repository.py#L48). You'll need to increment next_version after this INSERT.